src/components/ods-tables/table-basic/ods-table-basic.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component, Host, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'ods-table-basic',
+  styleUrl: 'ods-table-basic.scss',
+  shadow: true
+})
+
+export class ODSTableBasic {
```

Comment:
There really needs to be several components:
Table
TableHeader
TableHeaderCell
TableRow
TableCell
TableBody

Reply:
So, in this case this component would just return the slot and these other components would be defined separately.

```HTML
<Host>
  <table>
    <slot/>
  </table>
</Host>
```